code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -114,6 +114,23 @@
 		oogas = rand(2,6)
 		playsound(src, 'sound/creatures/gorilla.ogg', 50)
 
+/mob/living/simple_animal/hostile/gorilla/lesser
+	name = "lesser Gorilla"
+	desc = "An adolescent Gorilla. It may not be fully grown but, much like a banana, that just means it's all the more sturdier and hard to chrew through for it!"
```

Comment:
```suggestion
	desc = "An adolescent Gorilla. It may not be fully grown but, much like a banana, that just means it's sturdier and hard to chew!"
```